### PR TITLE
Adds CLF specialist to CLF ERT, and 2 guns for it.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds CLF specialist to CLF ERT.
Adds 2 guns which are the PTR AT rifle and the DP MG (Both sprites by crow, other than the DP inhands which are mine.)
## Why It's Good For The Game
Adds more interesting stuff to CLF, and interesting guns. The ERT in general probably needs some love.
## Changelog
:cl:
add: Added new CLF specialist role to the CLF ERT, and 2 new guns for it.
/:cl:

### CODE REVIEW

**Comment by mc-oofert** [comment]
PICTURES

**Review by Kuro020 — APPROVED** [review]

File: code/modules/projectiles/ammunition.dm [hunk 1]
```diff
@@ -560,6 +560,14 @@ Turn() or Shift() as there is virtually no overhead. ~N
 	base_icon_state = "ammoboxflechette"
 	ammo_type = /datum/ammo/bullet/shotgun/flechette
 
+/obj/item/shotgunbox/clf_heavyrifle
+	name = "14.5mm API Ammo Box"
```
Comment (XSlayer300) [inline code/modules/projectiles/ammunition.dm:564]:
```suggestion
	name = "big ammo box (14.5mm API)"
```
idc if the shotgun ammo box isn't consistent with the rest, go with the format

File: code/modules/projectiles/guns/rifles.dm [hunk 2]
```diff
@@ -671,6 +671,48 @@
 		/obj/item/attachable/foldable/bipod,
 	)
 
+//-------------------------------------------------------
+//DP-27
+
+/obj/item/weapon/gun/rifle/dpm
+	name = "\improper Degtyaryov 'RP' Machine Gun"
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/rifles.dm:678]:
```suggestion
	name = "\improper Degtyaryov 'RP' machine gun"
```

File: code/modules/projectiles/guns/rifles.dm [hunk 3]
```diff
@@ -671,6 +671,48 @@
...
+	desc = "A cheap and robust machine gun seen commonly in the fringes of the 'Bubble' it fires high caliber rounds to accomdate for a sluggish rate of fire, is generally found being called 'The Record Player' due to the resemblance. Fires 7.62x54mmR rounds."
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/rifles.dm:679]:
```suggestion
	desc = "A cheap and robust machine gun seen commonly in the fringes of the eastern bubble. Fires high caliber rounds to accomodate for its sluggish rate of fire, it is generally found being called 'The Record Player' due to the resemblance. Fires 7.62x54mmR rounds."
```

**Comment by Pariah919** [comment]
gonna reopen this when i figure a few things out.

**Pull request closed without merging**